src/app/modules/home/home.view.html

```diff
@@ -14,7 +14,7 @@
     <a class="btn btn-primary btn-lg btn-block"
        href="#!/players"
        role="button">
-      gioca
+      play
```

Comment:
La traduzione in inglese fatta in questo modo non migliora in alcun modo l'applicazione. L'internazionalizzazione delle applicazioni prevede l'uso di token e di un servizio atto a intercettare la lingua usata dall'utente.